modules/brainbrowser/README.md

```diff
@@ -0,0 +1,37 @@
+# Brainbrowser
+
+## Purpose
+
+Brainbrowser is intended to allow users to visualize candidate scan session MINC 
+volumes collected for a study.
+
+## Intended Users
+
+The two primary types of users are:
+- Imaging specialists doing online QC
+- Project radiologists reporting incidental findings.
+
+
+## Scope
+
+Brainbrowser allows navigating volumes in sagittal, coronal, and transverse 
+slices. Multiple volumes can be displayed simultaneously and coordinates
+synchronized across the multiple volumes, if desired. Image brightness and 
+intensity can be set by the user.
+A `Play` button is also available to make feasible the navigation through the 
+time dimension of a 4-Dimensional image (such as resting state fMRI scan). 
```

Comment:
I rewrote a little bit the section above moving around sentences:

"Brainbrowser allows visualization of MINC images directly within the browser. The user can navigate through those images across their sagittal, coronal and transverse axes. In addition, a `Play` button is also available for 4-Dimensional datasets to allow navigation through their fourth dimension (such as the time dimension for resting state fMRI scans). 
Multiple volumes can be displayed simultaneously with the possibility to synchronize all volumes' navigation when moving the cursor across one of the volume"

Reply:
I will use mostly what you suggested with some modifications (for example axes is in fact slices, ...).
Check the latest commit and let me know!

Reply:
maybe also mention DTI for a non-temporal 4th dimension example.